cypress/cypress/support/test-initializer/types.ts

```diff
@@ -0,0 +1,27 @@
+enum AuthType {
+    Cognito = 'COGNITO',
+    Idp = 'IDP',
```

Comment:
This is a little weird because `IdP` stands for Identity Provider of which Cognito would be one option. Assuming `Idp` in this case is the mock Idp then we should probably just remove it - the mock IdP isn't something that will be used moving forward and isn't something that is of use external to our team. The same comment aplies to the `mockIdp.ts` provider and any other references in this PR.

Reply:
I don't really think it makes sense as that's not something that will be maintained or available externally.